ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessRewriter.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.restmodel.displayedgraph
+
+import pl.touk.nussknacker.engine.graph.node.{Disableable, NodeData}
+import pl.touk.nussknacker.restmodel.displayedgraph.rewriter.{RemoveAndBypassNode, RemoveBranch}
+
+object ProcessRewriter {
+
+  def removeAndBypassNode(process: DisplayableProcess, node: NodeData): DisplayableProcess =
+    new RemoveAndBypassNode(process, node).apply
+
+  def removeBranch(process: DisplayableProcess, branchRoot: NodeData): DisplayableProcess =
+    new RemoveBranch(process, branchRoot).apply
+
+  def removeDisabledNodes(process: DisplayableProcess): DisplayableProcess =
+    process.nodes.collectFirst { case d: Disableable => d } match {
+      case Some(node) if node.isDisabled.getOrElse(false) =>
```

Comment:
1. Why not to merge both checks: `collectFirst { case d: Disableable if d.isDisabled ...`
2. Can you replace this with `contains(true)`? It is IMO less brain-damaging.